cloudmarker/util.py

```diff
@@ -101,6 +103,7 @@ def load_plugin(plugin_config):
 
     # Construct the plugin.
     plugin = plugin_class(**plugin_params)
+    plugin.plugin = plugin_config['plugin']
```

Comment:
Not sure if this line is required.

Reply:
This is not required indeed. This is a remnant from a detour I had taken to try out some ideas while working on this. I will remove this and update this pull request before merging this. Thank you for noticing this and pointing this out.

Reply:
Fixed this and force-pushed the updated branch in this pull request. Anyone can merge this as soon as the Travis CI build jobs succeed.